### ISSUE TITLE
Can't update platform

### ISSUE THREAD

**Issue by Miguelcldn** [Fermat-ORG/fermat-org#581]
By trying to rename WPD to APD I got the following error:
![pasted image at 2016_05_04 11_01 am](https://cloud.githubusercontent.com/assets/5544266/15021359/5a53c5d0-11f4-11e6-9a80-3a743fa82d36.png)

Here is the query data:
```
PUT /v1/repo/usrs/56c257f83e71b74f20aa897a/platfrms/56e1e2678fb3968867d28b66?axs_key=56e2209e067cc98e6488144e HTTP/1.1
Host: api.fermat.org
Cache-Control: no-cache
Postman-Token: d8c1de6a-3023-08c8-acc3-12ce33586a0a
Content-Type: application/x-www-form-urlencoded

code=APD&name=app+production+and+distribution&logo=APD_logo.png
```

**Comment by Miguelcldn** [comment]
Happens with Layers too

**Comment by campol** [comment]
No tienes token asignado, logueate en la app para que se te genere uno

**Comment by Miguelcldn** [comment]
![image](https://cloud.githubusercontent.com/assets/5544266/15433293/c23d4602-1e7f-11e6-8136-33956fea1a44.png)

Igual, no hay respuesta

**Comment by Miguelcldn** [comment]
Fixed, erro existed due #605
